src/patternfly/components/Popover/popover.scss

```diff
@@ -45,32 +41,37 @@
   --#{$popover}__arrow--m-left--TranslateX: 50%;
   --#{$popover}__arrow--m-left--TranslateY: -50%;
   --#{$popover}__arrow--m-left--Rotate: 45deg;
+  --#{$popover}__arrow--m-inline-top--Top: var(--pf-t--global--border--radius--medium);
+  --#{$popover}__arrow--m-inline-bottom--Bottom: var(--pf-t--global--border--radius--medium);
+  --#{$popover}__arrow--m-block-left--Left: var(--pf-t--global--border--radius--medium);
+  --#{$popover}__arrow--m-block-right--Right: var(--pf-t--global--border--radius--medium);
```

Comment:
Not 100% sure how I feel about the naming so open to other ideas. Could also just apply the border radius token directly where the top/right/bottom/left vars are reassigned rather than adding new css vars, but since we may want to do something similar in Tooltip it would make sense to do the same here.